tests/core/pyspec/eth2spec/test/helpers/sync_committee.py

```diff
@@ -153,6 +154,7 @@ def _build_block_for_next_slot_with_sync_participation(spec, state, committee_in
             state,
             block.slot - 1,
             [index for index, bit in zip(committee_indices, committee_bits) if bit],
+            block_root=block.parent_root,
```

Comment:
passing in block root if we have it, to avoid accidentally re-computing via an entirely new scratch block